### PR TITLE
fixed tap reporter changes

### PR DESCRIPTION
**Pull request by jameslnewell**

### CODE REVIEW

**Comment by jameslnewell** [comment]
classic "works on my machine" now.

File: test/phantom-test.js [hunk 1]
```diff
@@ -33,7 +33,6 @@ describe('phantom', function () {
       assert.equal(stdout.indexOf('# phantomjs:\n'
         + '1..1\n'
         + 'not ok 1 test fails\n'
-        + '  Error: Oh noes!\n'
```
Comment (mantoni) [inline test/phantom-test.js]: This change fails on travis and also for me locally. The other newline changes are fixing it for me as well.

Comment (jameslnewell) [inline test/phantom-test.js]: That's weird. It was broken for me on OS X with mocha@2.2.5 installed. What version do you have installed? Feel free to revert.

Comment (mantoni) [inline test/phantom-test.js]: I just ran `make install` which fetches the latest.

**Comment by mantoni** [comment]
Build was fixed in b9e03f4862270b4a7f7d3088eea5f10793569926

**Pull request closed without merging**